code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -566,3 +578,97 @@
 			G.affecting.forceMove(locate(T.x + rand(-1,1), T.y + rand(-1,1), T.z))
 		else
 			qdel(G)
+
+/mob/living/carbon/human/proc/empath()
+	set category = "Abilities"
+	set name = "Empath Reading"
+	set desc = "Glean from the minds of others."
+
+
+	var/obj/item/organ/external/rhand = src.get_organ("r_hand")
+	var/obj/item/organ/external/lhand = src.get_organ("l_hand")
+	if((!rhand || !rhand.is_usable()) && (!lhand || !lhand.is_usable()))
+		src << "<span class='warning'>You can't use your power without the ability to use your hands!</span>"
+		return FALSE
+	if(stat || paralysis || stunned || weakened ||  restrained())
+		src << "<span class='warning'>You can't use your power while unable to move your hands to your head!</span>"
+		return FALSE
+
+	visible_message("<span class='notice'>[src] touches their fingers to their temple.</span>")
+
+	var/list/targets = list()
+	var/target = null
+	var/text = null
+
+	targets += getmobs()
```

Comment:
So, they can read any mob in the world? This list can be used to meta and should never be exposed to an in-round player.